src/components/Firmware/V3UpdateTable.tsx

```
@@ -0,0 +1,289 @@
+                {component.latestVersion}
+              </label>
+            </div>
+          </div>
+        </td>
+        <td className="px-3 py-4 text-sm text-gray-800">
+          {component.changelog &&
+            (component.changelog.length < 100 ? (
+              <div
+                className="changelog-content"
+                // eslint-disable-next-line react/no-danger
+                dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)